sdk/cosmos/azure-cosmos-spark_3-0_2-12/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+## Release History
+
+## 4.0.0-beta.1 (Unreleased)
```

Comment:
NIT: We should have ### (3 hashtags here) - so that it can be directly used in the changelog. We can do this once we fill these changelogs. No need to block the PR.

Reply:
Fixed